bitnami/grafana-mimir/values.yaml

```diff
@@ -0,0 +1,4185 @@
+##
+commonAnnotations: {}
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+## @param extraDeploy Array of extra objects to deploy with the release
+##
+extraDeploy: []
+
+## Enable diagnostic mode in the deployment
+##
+diagnosticMode:
+  ## @param diagnosticMode.enabled Enable diagnostic mode (all probes will be disabled and the command will be overridden)
+  ##
+  enabled: false
+  ## @param diagnosticMode.command Command to override all containers in the deployment
+  ##
+  command:
+    - sleep
+  ## @param diagnosticMode.args Args to override all containers in the deployment
+  ##
+  args:
+    - infinity
+
+## @section Common Grafana Mimir Parameters
+##
+
+mimir:
+  ## Bitnami Grafana Mimir image
+  ## ref: https://hub.docker.com/r/bitnami/grafana-mimir/tags/
+  ## @param mimir.image.registry Grafana Mimir image registry
+  ## @param mimir.image.repository Grafana Mimir image repository
+  ## @param mimir.image.tag Grafana Mimir image tag (immutable tags are recommended)
+  ## @param mimir.image.digest Grafana Mimir image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag image tag (immutable tags are recommended)
+  ## @param mimir.image.pullPolicy Grafana Mimir image pull policy
+  ## @param mimir.image.pullSecrets Grafana Mimir image pull secrets
+  ## @param mimir.image.debug Enable Grafana Mimir image debug mode
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/grafana-mimir
+    tag: 2.5.0-debian-11-r6
+    digest: ""
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+    pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    pullSecrets: []
+    ## Enable debug mode
+    ##
+    debug: false
+  ## @param mimir.dataDir path to the Mimir data directory
+  ##
+  dataDir: "/bitnami/grafana-mimir"
+  ## @param mimir.configuration [string] Mimir components configuration
+  ##
+  configuration: |
+    usage_stats:
+      installation_mode: helm
+    activity_tracker:
+      filepath: {{ .Values.mimir.dataDir }}/activity.log
+    alertmanager_storage:
+      {{- if .Values.minio.enabled }}
+      backend: s3
+      s3:
+        access_key_id: ${MIMIR_MINIO_ACCESS_KEY_ID}
+        secret_access_key: ${MIMIR_MINIO_SECRET_ACCESS_KEY}
+        bucket_name: alertmanager
+        endpoint: "{{- include "grafana-mimir.minio.fullname" . -}}:{{ .Values.minio.service.ports.api }}"
+        insecure: {{ not .Values.minio.tls.enabled }}
+      {{- else }}
+      backend: {{ .Values.alertmanager.blockStorage.backend }}
```

Comment:
It is not straightforward the fact that blockstorage.backend will be ignored if `minio.enabled` is true. Maybe document it in the values.yaml?